src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx

```diff
@@ -691,13 +691,13 @@
     <value>Tydzień {0}</value>
   </data>
   <data name="Week1ScoreDescription" xml:space="preserve">
-    <value>Wyniki dla pierwszego tygodnia poznasz w niedzielę, 8 grudnia po godzinie 20:00.</value>
+    <value>Wyniki dla pierwszego tygodnia poznasz w niedzielę, 6 grudnia po godzinie 13:00.</value>
```

Comment:
8 December was correct - after 7 days. But the day of the week was incorrect (Tuesday)